erpnext/buying/report/procurement_tracker/procurement_tracker.py

```diff
@@ -207,7 +207,8 @@ def get_mapped_mr_details(conditions):
 			child.qty,
 			child.item_code,
 			child.uom,
-			par.status
+			par.status,
+			child.project
```

Comment:
change the `par` to `parent` please
Also, correct me if im wrong, project fetching is required for PO,PR,PI too right ?